### PR TITLE
Completion of sgt-back-end challenge. -Euichang Park

### PR DESCRIPTION
**Pull request by EuichangPark**

### CODE REVIEW

File: sgt-back-end/index.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const middleware = express.json();
+app.use('/api/grades', middleware);
```
Comment (scbowler) [inline sgt-back-end/index.js:14]: When applying the json middleware we want to apply it to all paths not just the specific path of `/api/grades`. This should be:
```JAVASCRIPT
app.use(middleware);
```

File: sgt-back-end/index.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+app.get('/api/grades', (req, res) => {
+  const sql = `
+  select "gradeId",
+         "name",
+         "course",
+         "score",
+         "createdAt"
+    from "grades"
+  `;
+  console.log('Hi');
```
Comment (scbowler) [inline sgt-back-end/index.js:25]: Remember to cleanup console logs before submitting your work for review.

File: sgt-back-end/index.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    db.query(sql, params)
+      .then(result => {
+        if (result.rows.length === 0) {
+          res.status(404).json({
+            error: `grade with id ${id} does not exist`
+          });
+        } else {
+          const grade = result.rows[0];
+          res.status(200).json(grade);
+        }
```
Comment (scbowler) [inline sgt-back-end/index.js:105]: If you rearrange this a bit you can clean it up some:
```JAVASCRIPT
const grade = result.rows[0];

if (!grade) {
  res.status(404).json({
    error: `grade with id ${id} does not exist`
  });
} else {         
  res.status(200).json(grade);
}
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just a few cleanup items needed. Let me know if you have any questions.

File: sgt-back-end/index.js (hunk 3 above)
Comment (EuichangPark) [reply sgt-back-end/index.js:105]: Oh wow...

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**